pt-BR/mail/chrome/messenger/addressbook/abMainWindow.dtd

```diff
@@ -119,41 +119,41 @@
 <!ENTITY preferencesCmd.key                             "O">
 <!ENTITY preferencesCmd.accesskey                       "O">
 <!ENTITY preferencesCmdUnix.label                       "Preferências">
-<!ENTITY preferencesCmdUnix.accesskey                   "P"> 
+<!ENTITY preferencesCmdUnix.accesskey                   "P">
 
 <!-- Help Menu -->
-<!ENTITY helpMenu.label                                 "Help">
+<!ENTITY helpMenu.label                                 "Ajuda">
 <!ENTITY helpMenu.accesskey                             "H">
-<!ENTITY releaseCmd.label                               "Release Notes">
+<!ENTITY releaseCmd.label                               "Notas da versão">
 <!ENTITY releaseCmd.accesskey                           "R">
-<!ENTITY hintsAndTips.label                             "&brandShortName; Help">
+<!ENTITY hintsAndTips.label                             "Ajuda do &brandShortName;">
 <!ENTITY hintsAndTips.accesskey                         "H">
 <!ENTITY hintsAndTipsMac.commandkey                     "?">
-<!ENTITY aboutCmd.label                                 "About &brandShortName;">
+<!ENTITY aboutCmd.label                                 "Sobre o &brandShortName;">
 <!ENTITY aboutCmd.accesskey                             "A">
 
-<!-- Address Book Toolbar and Context Menus -->  
-<!ENTITY newcardButton.label                            "New Card">
+<!-- Address Book Toolbar and Context Menus -->
+<!ENTITY newcardButton.label                            "Novo cartão">
 <!ENTITY newcardButton.accesskey                        "C">
 <!ENTITY newlistButton.label                            "Nova lista">
 <!ENTITY newlistButton.accesskey                        "l">
-<!ENTITY editButton.label                               "Properties">
+<!ENTITY editButton.label                               "Propriedades">
 <!ENTITY editButton.accesskey                           "P">
 <!ENTITY newmsgButton.label                             "Nova msg">
 <!ENTITY newmsgButton.accesskey                         "N">
 <!ENTITY newIM.label                                    "Msg instantânea">
 <!ENTITY newIM.accesskey                                "M">
-<!ENTITY deleteButton.label                             "Delete">
+<!ENTITY deleteButton.label                             "Excluir">
 <!ENTITY deleteButton.accesskey                         "D">
-<!ENTITY throbberItem.title                             "Activity Indicator">
+<!ENTITY throbberItem.title                             "Indicador de atividade">
 
 <!-- Address Book Toolbar Tooltips -->
-<!ENTITY newcardButton.tooltip                          "Create a new address book card">
-<!ENTITY newlistButton.tooltip                          "Criar uma nova lista">
-<!ENTITY editButton.tooltip                             "Edit the selected card">
+<!ENTITY newcardButton.tooltip                          "Criar um novo cartão no catálogo de endereços">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY newcardButton.tooltip                          "Adicionar um novo cart de visit">